### PR TITLE
WPF-856620: Update the tab key press description in PropertyGrid's UG documentation in WPF.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Update the tab key press description content in the PropertyGrid's UG documentation in WPF.

### CODE REVIEW

File: wpf/PropertyGrid/Keyboard-Navigation-support.md [hunk 1]
```diff
@@ -52,16 +52,27 @@ The following table explains how the navigation performed between properties,
 <tr>
 <td>7</td>
 <td>Tab</td>
-<td>When the PropertyName field is focused then the focus will be moved to value field and for next {{'**Tab**'| markdownify }} key press, focus will move to next property Name field from current property Value field.
+<td>Selection will be moved to the first item when the PropertyGrid doesn't have a selected item. It moves out of the PropertyGrid on the next Tab key when the focus is within the PropertyGrid.
 <br/>
```
Comment (Manivannan-E) [inline wpf/PropertyGrid/Keyboard-Navigation-support.md:56]: @Priyanka-Vijayasankar Selecting the first item when no item is chosen and moving out of the PropertyGrid on subsequent Tab key presses when the focus is already within.

File: wpf/PropertyGrid/Keyboard-Navigation-support.md [hunk 2]
```diff
@@ -52,16 +52,27 @@ The following table explains how the navigation performed between properties,
...
+<img src="KeyNavigation-Images/First-Tab-Key-Navigation.png" alt="Pressing the Tab key focuses on the first item"/>
 <br/>
 <br/>
 </td>
 </tr>
 <tr>
 <td>8</td>
+<td>Shift + Tab</td>
+<td>Selection will be moved to the last item when the PropertyGrid doesn't have a selected item. It moves out of the PropertyGrid on the next Shift + Tab key when the focus is within the PropertyGrid.
+<br/>
```
Comment (Manivannan-E) [inline wpf/PropertyGrid/Keyboard-Navigation-support.md:67]: @Priyanka-Vijayasankar, Selecting the last item when no selection exists, while subsequent Shift + Tab presses moves the focus out of the PropertyGrid when the focus is within.

File: wpf/PropertyGrid/Keyboard-Navigation-support.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply wpf/PropertyGrid/Keyboard-Navigation-support.md:56]: Updated the above mentioned content.

File: wpf/PropertyGrid/Keyboard-Navigation-support.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply wpf/PropertyGrid/Keyboard-Navigation-support.md:67]: Updated the above mentioned content.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**